cli/preset/cloud_preset.go

```diff
@@ -21,19 +21,24 @@ import (
 )
 
 const (
-	secretNameHCPConfig      = "consul-hcp-config"
-	secretNameGossipKey      = "consul-gossip-key"
-	secretNameBootstrapToken = "consul-bootstrap-token"
-	secretNameServerCA       = "consul-server-ca"
-	secretNameServerCert     = "consul-server-cert"
-	secretKeyHCPClientID     = "client-id"
-	secretKeyHCPClientSecret = "client-secret"
-	secretKeyHCPResourceID   = "resource-id"
-	secretKeyHCPAuthURL      = "auth-url"
-	secretKeyHCPAPIHostname  = "api-hostname"
-	secretKeyHCPScadaAddress = "scada-address"
-	secretKeyGossipKey       = "key"
-	secretKeyBootstrapToken  = "token"
+	secretNameHCPClientID     = "consul-hcp-client-id"
```

Comment:
Nit: I would have organized these by name/key pairs so that it is easier to tell if any of them are missing.